File: src/main/java/me/centralhardware/znatoki/telegram/statistic/configuration/ResourceConfiguration.java
package me.centralhardware.znatoki.telegram.statistic.configuration;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.util.Locale;
import java.util.ResourceBundle;

@Configuration
public class ResourceConfiguration {

    public static final String RESOURCE_BUNDLE_NAME = "Strings";

    @Bean
    public ResourceBundle getResource() {
        return ResourceBundle.getBundle(RESOURCE_BUNDLE_NAME, Locale.US);
    }

}

File: src/main/kotlin/me/centralhardware/znatoki/telegram/statistic/configuration/ResourceConfiguration.kt
package me.centralhardware.znatoki.telegram.statistic.configuration

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.util.*

@Configuration
class ResourceConfiguration {

    companion object {
        const val RESOURCE_BUNDLE_NAME = "Strings"
    }

    @Bean
    fun getResource(): ResourceBundle {
        return ResourceBundle.getBundle(RESOURCE_BUNDLE_NAME, Locale.US)
    }

}